test/test.py

```diff
@@ -129,7 +132,11 @@ def testClient(self):
                 self.assertEqual(len(subset), docs.total)
                 ids = [x.id for x in docs.docs]
                 self.assertEqual(set(ids), set(subset))
-                
+
+                for doc in client.search(Query('henry king').returning('play', 'nonexist')).docs:
+                    doc.nonexist
```

Comment:
This is deliberate. I'm unsure what exactly would be included in the field, but I could be sure that trying to access this field would *not* result in an exception (e.g. AttributeError)